### PR TITLE
make 0-3 examples work for Vereya

### PR DESCRIPTION
**Pull request by DaddyWesker**

### CODE REVIEW

File: examples/1_startandwait.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from tagilmo.utils.malmo_wrapper import MalmoConnector
+import tagilmo.utils.mission_builder as mb
+
+
+miss = mb.MissionXML()
+# just an empty flat world for testing
+miss.setWorld(mb.flatworld(""))
```
Comment (Necr0x0Der) [inline examples/1_startandwait.py:7]: Doesn't work for me. I'd use the default world here, but with FIXME

File: examples/3_command.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            else:
+                mc.sendCommand("turn 0")
+            mc.sendCommand("move 1")
+        sleep(0.1)
+        print("Done")
+        mc.sendCommand("move 0")
+        mc.sendCommand("turn 0")
+
+miss = mb.MissionXML()
+# miss.setWorld(mb.flatworld("3;7,25*1,3*3,2;1;stronghold,biome_1,village,decoration,dungeon,lake,mineshaft,lava_lake"))
```
Comment (Necr0x0Der) [inline examples/3_command.py:47]: This script is not intended for the default world, so I'd add a FIXME comment here (without changing the code)

File: examples/1_startandwait.py (hunk 1 above)
Comment (DaddyWesker) [reply examples/1_startandwait.py]: Weird, that works for me. Anyway, done.

File: examples/3_command.py (hunk 2 above)
Comment (DaddyWesker) [reply examples/3_command.py:49]: Done

**Pull request merged**